lib/buildTool/protractor/testBrowser/templates/protractor.conf.js.tpl

```diff
@@ -58,9 +58,10 @@ var config = {
 
 
   onPrepare: function() {
+    <% if (buildJS.framework[0] && buildJS.framework[0].toLowerCase().search('angular') === -1) { %>
```

Comment:
I think the tests failed because the framework names are in title case (see https://github.com/odecee/generator-confit/blob/master/lib/projectType/browser/browserResources.yml#L150-L202)